sdk/core/core-amqp/src/retry.ts

```diff
@@ -67,13 +72,29 @@ export interface RetryConfig<T> {
   /**
    * @property {number} [delayInSeconds] Amount of time to wait in seconds before making the
    * next attempt. Default: 15.
+   * When `exponentialRetry` is set to `true`, this is used to compute the exponentially increasing delays between retries.
```

Comment:
exponentialRetry -> isExponentialRetry

Same in a few other places where the old name is being used in comments

Reply:
Addressed, re-addressed :)